core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/QC.java

```diff
@@ -104,4 +104,12 @@ public void setResults(Double results) {
   public void setType(QcType type) {
     this.type = type;
   }
+
+  @Override
+  public int compareTo(QC o) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1210)